docs/mint.json

```diff
@@ -264,6 +264,7 @@
             "backends/elasticsearch",
             "backends/gigapipe",
             "backends/googlecloudmonitoring",
+            "backends/googlecloudotlp",
```

Comment:
wonder if we should also add it [here](https://github.com/odigos-io/odigos/blob/main/docs/backends-overview.mdx) for completeness. even though it is a special case (same product, different protocol)

Reply:
didn't know about that table, yeah I think so